runtime/runtime-params-estimator/src/cases.rs

```diff
@@ -35,26 +36,79 @@ fn measure_function(
     metric: Metric,
     method_name: &'static str,
     measurements: &mut Measurements,
-    testbed: RuntimeTestbed,
+    testbed: Arc<Mutex<RuntimeTestbed>>,
     accounts_deployed: &[usize],
     nonces: &mut HashMap<usize, u64>,
     config: &Config,
     allow_failures: bool,
     args: Vec<u8>,
-) -> RuntimeTestbed {
+) -> Arc<Mutex<RuntimeTestbed>> {
     // Measure the speed of creating a function fixture with 1MiB input.
     let mut rng = rand_xorshift::XorShiftRng::from_seed([0u8; 16]);
+    let testbed = testbed.clone();
+    let mut accounts_used = HashSet::new();
     let mut f = || {
-        let account_idx = *accounts_deployed.choose(&mut rng).unwrap();
+        let account_idx = loop {
+            let i = rand::thread_rng().gen::<usize>() % accounts_deployed.len();
+            let x = accounts_deployed[i];
+            if accounts_used.contains(&x) {
+                continue;
+            }
+            break x;
+        };
+        accounts_used.insert(account_idx);
```

Comment:
Since `accounts_deployed` is fairly small, we should instead clone it and then draw and remove from it.

Reply:
Fixed